<?xml version="1.1"?>
<launch>
    <arg name="pointcloud_topic" default="radar_points"/>

    <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan" output="screen">
        <param name="pointcloud_topic" value="$(arg pointcloud_topic)"/>
        <param name="min_height" value="-0.5"/>
    </node>

</launch>
